ddtrace/internal/runtime/tag_collectors.py

```diff
@@ -24,8 +24,8 @@ def collect_fn(self, keys):
         ddtrace = self.modules.get("ddtrace")
         # make sure to copy _services to avoid RuntimeError: Set changed size during iteration
         tags = [(SERVICE, service) for service in list(ddtrace.tracer._services)]
-        if ENV_KEY in ddtrace.tracer.tags:
-            tags.append((ENV_KEY, ddtrace.tracer.tags[ENV_KEY]))
+        if ddtrace.tracer._tags.get(ENV_KEY):
```

Comment:
This is subtly different behaviour for falsy values.

Can we just use the same check but use the internal attribute?